data/battle/effect_command_pointers.asm

```diff
@@ -180,5 +180,9 @@ BattleCommandPointers:
 	dw BattleCommand_SuperEffectiveLoopText
 	dw BattleCommand_StartLoop
 	dw BattleCommand_Curl
+	;==========================
+	;	New Moves Start Here
+	;==========================
+	dw BattleCommand_StartHail
 	dw BattleCommand_LowKick
```

Comment:
lowkick not a new move

Reply:
Would changing the wording to "New Effects Start Here" be sufficient? I believe that's what initially intended.

Reply:
Honestly... i'd just remove the comments altogether... when we start gen4.. would we have.. `new new moves start here`?

Doesn't really serve a purpose.

Reply:
It was just to separate vanilla is all, but I can go and remove them. I'm not married to them.

Reply:
Comments have been removed.